packages/adapter-auto/adapters.js

```diff
@@ -24,5 +24,11 @@ export const adapters = [
 		test: () => process.env.GITHUB_ACTION_REPOSITORY === 'Azure/static-web-apps-deploy',
 		module: 'svelte-adapter-azure-swa',
 		version: '0.13'
+	},
+	{
+		name: 'Deta',
+		test: () => !!process.env.DETA_PROJECT_KEY,
+		module: 'svelte-adapter-node',
+		version: '1.1.8'
```

Comment:
```suggestion
		module: '@sveltejs/adapter-node',
		version: '1'
```

Reply:
It seems `deta` is something like a docker deployment which they call a personal cloud.
So each deployment gets the same production port (probably `4173`) which they already handle internally when the adapter-node is used.

Using the environment variable seems like a good alternative in that case we won't need this change.